met-api/src/met_api/models/submission.py

```diff
@@ -72,6 +84,31 @@ def create(cls, submission: SubmissionSchema, session=None) -> Submission:
             session.flush()
         return new_submission
 
+    @staticmethod
```

Comment:
```
@staticmethod
def __check_if_submission_has_comments(submission: SubmissionSchema):
    """Check if comment exists."""
    submission_json = submission.get('submission_json', {})
    text_fields = ['simpletextarea', 'simpletextfield']

    for field in text_fields:
        if field in submission_json and len(submission_json[field]) > 0:
            return True

    return False
```

Reply:
can we simply this?
do check the logic in detail please..i am not sure 100% if it works

Reply:
Changed the logic as suggested, thanks.